platforms/web/lib/composer.ts

```diff
@@ -71,6 +71,13 @@ export function processInput(
             return action(composerModel.backspace(), 'backspace');
         case 'deleteWordBackward':
             return action(composerModel.backspace_word(), 'backspace_word');
+        case 'deleteSoftLineBackward': {
+            document
+                .getSelection()
+                ?.modify('extend', 'backward', 'lineboundary');
+            document.dispatchEvent(new CustomEvent('selectionchange'));
```

Comment:
If we can't call modify (ie no selection), I think we shouldn't dispatch the `selectionchange` event

Reply:
That's a fair point, I'll do that to appease TS, but the case that returns a null selection is pretty niche

Reply:
Changes applied